src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.HandlePropertyName.cs

```diff
@@ -5,11 +5,14 @@
 using System.Buffers;
 using System.Collections;
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
 
 namespace System.Text.Json
 {
     public static partial class JsonSerializer
     {
+        // AggressiveInlining used although a large method it is only called from one locations and is on a hot path.
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
@stephentoub, @jkotas - Given our previous discussions on the use of this attribute, what are your thoughts on the use of `AggressiveInlining` in cases like these, where there are very few (1-2) callers of somewhat large methods, and benchmarks show improvements for adding them?